tests/conftest.py

```
@@ -1,190 +1,346 @@
+    """동기 FastAPI 테스트 클라이언트"""
+    return TestClient(app)
 
 
-# 테스트용 데이터베이스 세션 의존성 override
-async def get_test_session():
-    """테스트용 데이터베이스 세션 생성"""
-    global _test_engine, _test_session_maker
+@pytest_asyncio.fixture
+async def async_client() -> AsyncGenerator[AsyncClient, None]:
+    """비동기 HTTPx 클라이언트"""
     
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)